app/controllers/concerns/admin/list_paginated_users.rb

```diff
@@ -19,6 +19,8 @@ def list_paginated_users(users:, template:, legacy_template:)
         countless_minimal: true
       )
 
+      yield users if block_given?
```

Comment:
@binary-koan Reviewing this as I'm modelling my affiliates PR after this - won't this also redirect if we're on the last page, and that _page_ happens to only contain one user?

Reply:
That's correct. To fix this we can
- check for the users count before paginating (at the cost of one more query in a huge table), 
or
- check if it is the first page (`params[:page]` is nil or 1 or yield `pagination.page` and check if it is equal to 1)

Reply:
True, nice catch! Yielding pagination and checking the page sounds better to me, but up to you